scripts/README.md

```diff
@@ -38,6 +38,8 @@ In case of a problem, another method to generate the changelog is available at t
 2. Clean the generated changelog, to match the format of [https://github.com/mui/mui-x/releases](https://github.com/mui/mui-x/releases).
 3. Update the root `package.json`'s version
 4. Update the versions of the other `package.json` files and of the dependencies with `pnpm release:version` (`pnpm release:version prerelease` for alpha / beta releases).
+   Make sure to not skip the version bump if Lerna detected a change in the package. It is important to release the package if there were **any** changes to it.
+   If Lerna doesn't suggest a version bump for the package, we don't release it.
```

Comment:
Nitpick to avoid Vale error, but the existing wording works great as well. 👍 
```suggestion
   If Lerna doesn't suggest a version bump for the package, there is no need to release it.
```